contracts/end.cairo

```diff
@@ -764,7 +764,7 @@ func skim{
 
     // require(wad <= 2**255 && art <= 2**255, "End/overflow");
     with_attr error_message("End/overflow") {
-        let max =  Uint256(2 ** 128 - 1, 2 ** 128 - 1);
+        let max = Uint256(2 ** 128 - 1, 2 ** 128 - 1);
```

Comment:
Btw, Solidity version is wrong here, in a sense that it will revert for `wad == 2**255`, because `-int256(wad)` is a checked operation and will revert for `2**255 ` (`int256(2**255)` is actually `-2**255`, inversion of which is not an `int256`).
Solidity version should check: `wad <= type(int256).max`

Reply:
Thanks for that explanation @maciejka, do they plan to fix it tho ?

Reply:
They are aware of the problem.